Purchases/Public/Purchases.swift

```diff
@@ -261,6 +261,8 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
         let userDefaults = userDefaults ?? UserDefaults.standard
         let deviceCache = DeviceCache(systemInfo: systemInfo, userDefaults: userDefaults)
         let receiptParser = ReceiptParser()
```

Comment:
Just like `PurchasesOrchestrator`, I think `Purchases` no longer needs to keep a copy of `receiptParser` (it's not used anywhere?)

Reply:
yep, good catch 👍🏼